java/sage/media/format/SubpictureFormat.java

```diff
@@ -23,8 +23,8 @@
 {
   public String toString()
   {
-    return "Subpic[" + formatName + (id != null ? (" id=" + id) : "") + (path != null ? (" path=" + path) : "") +
-        (language == null ? "]" : (" " + language + "]"));
+    return "Subpic[" + formatName + (id != null ? (" id=" + id) : "") + (path != null ? (" path=" + path) : "")
+                + (language == null ? "" : (" " + language)) + " forced=" + forced + "]";
```

Comment:
Change where you have:

" forced=" + forced

to be

(forced ? " forced" : "")